src/unix/pipe.c

```diff
@@ -302,3 +302,16 @@ uv_handle_type uv_pipe_pending_type(uv_pipe_t* handle) {
   else
     return uv__handle_type(handle->accepted_fd);
 }
+
+int uv_pipe_chmod(uv_pipe_t* handle, int mode) {
+  struct stat pipe_stat;
+  if (fstat(handle->u.fd, &pipe_stat) == -1)
```

Comment:
I think this should be `handle->io_watcher.fd` or you could use `uv__stream_fd(handle)` directly